packages/flutter_tools/lib/src/run_hot.dart

```diff
@@ -425,11 +427,19 @@ class HotRunner extends ResidentRunner {
       final Uri devicePackagesUri = _devFS.baseUri.resolve('.packages');
       if (benchmarkMode)
         vmReloadTimer.start();
-      final Map<String, dynamic> reloadReport =
-          await currentView.uiIsolate.reloadSources(
+
+      Map<String, dynamic> reloadReport;
+      for (FlutterView view in currentViews) {
+        final Map<String, dynamic> report =
+            await view.uiIsolate.reloadSources(
```

Comment:
i'd remove the newline after the =
the remainder should only be indented by 2
if there's a newline after then ( please put a newline before the )